packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -14,6 +14,97 @@ import 'gesture_detector.dart';
 import 'navigator.dart';
 import 'transitions.dart';
 
+/// A widget that modifies the size of the [SemanticsNode.rect] (focus) of its
+/// child widget. It clips the focus in possibily four directions based on the
+/// specified [EdgeInsets].
+///
+/// If a [ValueNotifier] is provided, the size of the focus is adjusted based on
+/// value changes inside the [ValueNotifier].
+///
+/// If no [ValueNotifier] is provided, the [SemanticsClipper] applies a default
```

Comment:
I think we should force it to be non null. If this is null there is no reason to use this widget.

Reply:
Sounds good! Done.